### PR TITLE
feat: update preview to strapi 3.3.3 version

### PR DESCRIPTION
**Pull request by mstrag**

### CODE REVIEW

File: packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
+import React, { memo, useCallback, useMemo, useRef, useState } from "react";
```
Comment (mstrag) [inline packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/index.js:1]: diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files

File: packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/utils/connect.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
```
Comment (mstrag) [inline packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/utils/connect.js:1]: diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files

File: packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/ListView/index.js [hunk 3]
```diff
@@ -9,28 +9,29 @@ import React, {
 import PropTypes from "prop-types";
```
Comment (mstrag) [inline packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/ListView/index.js:9]: diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files

File: packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/index.js [hunk 4]
```diff
@@ -0,0 +1,303 @@
+import React, { memo, useCallback, useMemo, useRef, useState } from 'react';
```
Comment (mstrag) [inline packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/index.js:1]: diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files

File: packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/utils/connect.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (mstrag) [inline packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/utils/connect.js:1]: diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files

**Review by Kronos66 — APPROVED** [review]

**Review by cyp3rius — APPROVED** [review]

**Pull request merged**